Documentation/O3DEMPSProjectExportTesting.md

```diff
@@ -2,72 +2,139 @@
+    ```
+2. Clone the development branch of the O3DE Multiplayer Sample Assets and its PopcornFX submodule <br/>
+   <br/>
+    **Windows**
+    ```commandline
+    git clone -b development https://github.com/o3de/o3de-multiplayersample-assets.git %PROJECT_BASE%\o3de-multiplayersample-assets
+    git -C %PROJECT_BASE%\o3de-multiplayersample-assets submodule update --init --recursive
+    ```
+    
+    **Linux**
+    ```bash
+    git clone -b development https://github.com/o3de/o3de-multiplayersample-assets.git %PROJECT_BASE%/o3de-multiplayersample-assets
+    git -C $PROJECT_BASE/o3de-multiplayersample-assets submodule update --init --recursive
+    ```
+   > Note: You may need to check the git revision of the Popcorn FX Plugin under:
+   > 
+   >> ...\o3de-multiplayersample-assets\Gems\O3DEPopcornFXPlugin
+   > 
+   > to make sure it is the latest version.
 
-It is recommended to set the following environment variables with custom values according to your setup before running the export script:
-```bash
-# On Windows
-set O3DE_PATH="C:\path\to\o3de"
-set O3DE_PROJECT_PATH="C:\path\to\o3de-multiplayersample"
-set OUTPUT_PATH="C:\path\to\output"
-
-# On Linux
-export O3DE_PATH='/path/to/o3de'
-export O3DE_PROJECT_PATH='/path/to/o3de-multiplayersample'
-export OUTPUT_PATH='/path/to/output'
-```
+   <br/>
+3. Register the Gems path for the O3DE Multiplayer Sample Assets that is needed for the Multiplayer Sample<br/>
+   <br/>
+    **Windows**
+    ```commandline
+    %PROJECT_BASE%\o3de\scripts\o3de.bat register --all-gems-path %PROJECT_BASE%\o3de-multiplayersample-assets\Gems 
+    ```
+    **Linux**
+    ```bash
+    $PROJECT_BASE/o3de/scripts/o3de.sh register --all-gems-path $PROJECT_BASE/o3de-multiplayersample-assets/Gems 
+    ```
+   <br/>
 
-After setting those variables, navigate to that directory, and run the export command verbatim:
-```bash
+4. Clone the development branch of the O3DE Multiplayer Sample and register the project:<br/><br/>
 
-# On Windows
-%O3DE_PATH%\scripts\o3de export-project -es %O3DE_PATH%\scripts\o3de\ExportScripts\export_standalone_monolithic_project_centric.py -pp %O3DE_PROJECT_PATH% -out %OUTPUT_PATH% -cfg release -a zip -nounified -gpfp launch_client.cfg -spfp launch_client.cfg -code -assets -ll INFO -sl %O3DE_PROJECT_PATH%\AssetBundling\SeedLists\BasePopcornFxSeedList.seed -sl %O3DE_PROJECT_PATH%\AssetBundling\SeedLists\GameSeedList.seed -sl %O3DE_PROJECT_PATH%\AssetBundling\SeedLists\VFXSeedList.seed
+    **Windows**
+    ```commandline
+    git clone -b development https://github.com/o3de/o3de-multiplayersample.git %PROJECT_BASE%\o3de-multiplayersample
+    %PROJECT_BASE%\o3de\scripts\o3de.bat register -pp %PROJECT_BASE%\o3de-multiplayersample
+    ```
+    **Linux**
+    ```bash
+    git clone -b development https://github.com/o3de/o3de-multiplayersample.git $PROJECT_BASE/o3de-multiplayersample
+    $PROJECT_BASE/o3de/scripts/o3de.sh register -pp $PROJECT_BASE/o3de-multiplayersample
+    ```
 
-# On Linux
```

Comment:
I'm not sure how I feel about the removal of these export-project command invocations. I get the intent to consolidate, but the purpose of the readme was for testers to have some way of calibrating the export project feature in o3de engine with the current state of MPS.

Could it be added back somehow, at least as an option for folks that need to test not using `export_mps`?

Reply:
I'm not sure what you mean by 'removal of these export-project command invocations`. You do not have to use `export_mps`. You can still specify the `export_source_built_project.py` from the engine (by following the READM
E.md there).  The `export_mps` was updated from the original script to use the refactored methods in the export_project.py, and was updated to reflect the possibility of creating a custom export script for this project specifically.

Reply:
...oh i see it now, I think I misread what the intent of the revised instructions were. I'll resolve this comment, as there's no issue on my end now.